frontend/Components/Tickets/TicketsFilter.js

```diff
@@ -1,14 +1,17 @@
-import React, { useEffect } from 'react';
+import React, { useContext } from 'react';
 import { Typography } from '../Typography/Typography';
 import { Box } from '../Box/Box'
 import text from '../../theme/text';
 import colors from '../../theme/colors';
 import Select from '../Buttons/Select';
 import Button from '../Buttons/Button';
 import { ScrollView } from 'react-native';
+import { ModalContext } from '../../providers/ModalProvider';
 
 const TicketsFilter = (props) => {
   const { height } = props;
+  const { filter, setFilter } = useContext(ModalContext);
+
```

Comment:
@VasylMartyniv you can add data here as a state here for the modal (for before when allowing camera)